contentctl/actions/inspect.py

```diff
@@ -1,78 +1,88 @@
 import sys
-
-
 from dataclasses import dataclass
-
 import pathlib
 import json
 import datetime
+import timeit
+import time
 
-
-from contentctl.objects.config import inspect
 from requests import Session, post, get
 from requests.auth import HTTPBasicAuth
-import timeit
-import time
+
+from contentctl.objects.config import inspect
+from contentctl.objects.savedsearches_conf import SavedsearchesConf
+from contentctl.objects.errors import (
+    MetadataValidationError,
+    DetectionIDError,
+    DetectionMissingError,
+    VersionDecrementedError,
+    VersionBumpingError
+)
+
+
 @dataclass(frozen=True)
 class InspectInputDto:
-    config:inspect
+    config: inspect
 
 
+# TODO (cmcginley): how should detection version checking behave when it's run on just public, or
+#   just private content?
+# TODO (cmcginley): add `downloads/` to security_content .gitignore
 class Inspect:
 
     def execute(self, config: inspect) -> str:
-        if config.build_app or config.build_api:    
-         
+        if config.build_app or config.build_api:
+
             self.inspectAppCLI(config)
             appinspect_token = self.inspectAppAPI(config)
-            
-            
+
+            self.check_detection_metadata(config)
+
+            # TODO (cmcginley): do we need to return the appinspect token? Nothing is being done
+            #   with it
             return appinspect_token
```

Comment:
Do we need to return this token? It doesn't appear to be used anywhere outside this function

Reply:
Got it, that makes sense. I can decorate this with the comment explaining that if you want?